sydent/replication/peer.py

```diff
@@ -24,7 +23,7 @@
 import json
 
 import twisted.internet.reactor
-import twisted.internet.defer
+from twisted.internet import defer
```

Comment:
an unqualified `defer`, otoh, doesn't seem to appear at all.

Reply:
Ah, Python imports.